roles/common/tasks/main.yml

```diff
@@ -67,6 +67,20 @@
   notify:
     - reload logrotate timer settings
 
+- name: Common | Configure custom logrotate rules
+  ansible.builtin.template:
+    src: "logrotate_rules.j2"
+    dest: "/etc/logrotate.d/{{ item.name }}"
+    mode: "0644"
+    owner: root
+    group: root
+  loop: "{{ logrotate_rules }}"
```

Comment:
Is this task going to be used to set log rotation for the logs that appear on every machine? Do we want the custom rules to happen in the specific roles (rails, solr, etc.) rather than here?